prow/github/report/report.go

```diff
@@ -201,7 +224,7 @@ func parseIssueComments(pj prowapi.ProwJob, isBot func(string) bool, ics []githu
 		}
 		// Old report comments started with the context. Delete them.
 		// TODO(spxtr): Delete this check a few weeks after this merges.
```

Comment:
We'd need to check if the comment contained any context, not just the one from the first PR.

That being said, this comment is almost 5 years old. We can just remove the check rather than continuing to update it. Any existing Prow instances that have not been updated in 5 years can deal with some stray comments if they ever decide to update :upside_down_face:

Reply:
deleted